.coderabbit.yaml

```
@@ -0,0 +1,72 @@
+reviews:
+  # Enable automatic reviews
+  auto_review: true
+  
+  # Review focus areas
+  focus_areas:
+    - "security"
+    - "performance"
+    - "maintainability"
+    - "testing"
+    - "documentation"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)